### ISSUE TITLE
Increase spacing for headers and paragraphs

### ISSUE THREAD

**Issue by yannickdeturck** [ordina-jworks/ordina-jworks.github.io#37]
I would changes the spacing from:
```
h1, h2, h3, h4, h5, h6 {
    margin: 10px 0;
}
```

To:

```
h1, h2, h3, h4, h5, h6 {
    margin: 20px 0 10px 0;
}

p {
    margin: 10px 0;
}
```

**Comment by Turbots** [comment]
Also, make the text font a bit bigger:

```css
    .tk-acumin-pro {
        font-size: 130%;
    }
```

Then it looks much nicer!

**Comment by MrJean** [comment]
#39 is now open.

**Comment by yannickdeturck** [comment]
Fixed by 843cbb8.
